### PR TITLE
feat(ActivityCenter): Click on reply message jumps to message in chat log

### PR DESCRIPTION
**Pull request by borismelnik**
Fixes: #8121

### What does the PR do

Makes activity center reply clickable

### Affected areas

Activity Center

### Screenshot of functionality (including design for comparison)

https://user-images.githubusercontent.com/82511785/203430609-63a1f590-353e-4337-9e26-4b7209a76f9f.mov

### CODE REVIEW

**Comment by borismelnik** [comment]
@anastasiyaig check it pls is that what you want?

**Comment by anastasiyaig** [comment]
@borismelnik can we please implement this point too:

`clicking a message card from Mentions section in Activity center should open the channel and scroll history to that message`

it was working before but seems to be gone. Sorry it was not described initially in the ticket, but i added in in acceptance critria little bit later (u can find all of them in the issue, they all should work as described)

**Comment by borismelnik** [comment]
> @borismelnik can we please implement this point too:
> 
> `clicking a message card from Mentions section in Activity center should open the channel and scroll history to that message`
> 
> it was working before but seems to be gone. Sorry it was not described initially in the ticket, but i added in in acceptance critria little bit later (u can find all of them in the issue, they all should work as described)

done :)

**Comment by MishkaRogachev** [comment]
@borismelnik @anastasiyaig @iurimatias 
This functionality is already present but clicking on reply badge, not on whole message 
https://user-images.githubusercontent.com/2522130/203502034-23136218-4f60-478b-ad12-01d474007db0.mov

**Comment by anastasiyaig** [comment]
it is now aligned with designs @MishkaRogachev

**Review by anastasiyaig — APPROVED** [review]

File: ui/app/mainui/activitycenter/views/ActivityNotificationMessage.qml [hunk 1]
```diff
@@ -43,71 +45,77 @@ ActivityNotificationBase {
...
-                name: root.messageDetails.sender.displayName
-                asset: root.messageDetails.sender.profileImage.assetSettings
-                ringSettings: root.messageDetails.sender.profileImage.ringSettings
-
-                MouseArea {
-                    cursorShape: enabled ? Qt.PointingHandCursor : Qt.ArrowCursor
-                    acceptedButtons: Qt.LeftButton | Qt.RightButton
-                    anchors.fill: parent
-                    onClicked: root.openProfilePopup()
+    bodyComponent: MouseArea {
```
[the remark this answers is not in the document]
Comment (borismelnik) [reply ui/app/mainui/activitycenter/views/ActivityNotificationMessage.qml:48]: this kind of usage can make a problem if body have a `MouseArea` inside loader :) I can try to embed bodyLoader in MouseArea

**Comment by anastasiyaig** [comment]
any chance we have it merged?

**Pull request merged**